src/analytics.js

```diff
@@ -86,6 +87,9 @@ export class AnalyticsAddon extends AddonBase {
   }
 
   static isEnabled(config) {
-    return config.addons && config.addons.analytics.enabled === true;
+    const validate = ajv.getSchema(
+      "https://readthedocs.org/schemas/addons.analytics.json",
```

Comment:
I'm assuming this is a completely arbitrary string, and the URL probably doesn't matter, but I might suggest using a more obviously fake URL in case this somehow translates to a web request at some point -- or perhaps just to help communicate the URL doesn't matter at all.

No preference on what exactly that is, but I'm describing something like:

```suggestion
      "https://v1.schemas.readthedocs.org/addons.analytics.json",
```

Reply:
He, yeah. This URLs are dumb, but it's how the identifier works 🤷🏼 

> You don't have to host your schema files at the URIs that you use as schema $id. These URIs are only used to identify the schemas, and according to JSON Schema specification validators should not expect to be able to download the schemas from these URIs.

_(from https://ajv.js.org/guide/combining-schemas.html)_

The URI doesn't matter at all. I'm fine using whatever fake URL here 👍🏼